tee-worker/ts-tests/batch.test.ts

```diff
@@ -88,8 +98,13 @@ describeLitentry('Test Batch Utility', 0, (context) => {
             'identityManagement',
             ['IdentityCreated']
         );
-        const event_datas = await handleIdentityEvents(context, aesKey, resp_events, 'IdentityCreated');
-        for (let i = 0; i < event_datas.length; i++) {
+        const event_datas = (await handleIdentityEvents(
+            context,
+            aesKey,
+            resp_events,
+            'IdentityCreated'
+        )) as IdentityGenericEvent[];
+        for (let i = 0; i < [event_datas].length; i++) {
             assertIdentityCreated(context.substrateWallet.alice, event_datas[i]);
         }
```

Comment:
Isn't this the same as

```suggestion
        assertIdentityCreated(context.substrateWallet.alice, event_datas);
```

?

Reply:
It might be necessary to rewrite assertIdentityCreated to handle the data internally, as it seems more appropriate.